_content/papers/how-the-steps-of-mindfulness-of-breathing-decreased-from-sixteen-to-two_analayo.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "How the Steps of Mindfulness of Breathing Decreased from Sixteen to Two"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/howmindfulnessbreathing.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1d5w3Chqq2XnQ03NofFzcKAfTFSxrDaZb/view?usp=sharing"
+course: sati
+tags:
+  - anapanasati
+  - modern
+year: 2021
+publisher: "Centre of Buddhist Studies, University of Hong Kong"
+address: "Hong Kong"
+series: "HKU: CBS publication series"
+from_book: "Illuminating the Dharma: Buddhist Studies in Honour of Venerable Professor K.L. Dhammajoti"
```

Comment:
Sorry, missed it the first time (also confusing to me 😅) This should be "booktitle"